src/test/java/com/connexta/search/index/IndexServiceUnitTests.java

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
What is the default naming scheme? Is it `IndexServiceITest`, `IndexServiceComponentTest`,  and `IndexServiceTest`?

Reply:
AFAIK, yes.

Reply:
I think we need to have a larger conversation about the structure of our tests, because it it different than what we did in DDF. For now, I have updated the classes to follow these naming standards. Let me know if there is a better way to do this.
- -`ITest` when nothing is mocked (`@SpringBootTest`)
- -`ComponentTest` when beans are mocked (`@WebMvcTest(Xxx.class)`)
- -`Test` for unit tests